src/main/java/com/iexec/sms/admin/AdminService.java

```diff
@@ -284,16 +337,17 @@ private void checkDestinationFileNotExists(Path destination, AdminOperationError
     /**
      * Copy a file to another location with detailed trace information
      *
-     * @param source          The current location
-     * @param destination     The destination location
-     * @param fileDescription Short description associated with the file for better traceability
+     * @param source             The current location
+     * @param destination        The destination location
+     * @param fileDescription    Short description associated with the file for better traceability
+     * @param standardCopyOption Copy option
      * @throws IOException If the copy fails
      */
-    private void processCopyFile(Path source, Path destination, String fileDescription) throws IOException {
+    private void processCopyFile(Path source, Path destination, String fileDescription, StandardCopyOption standardCopyOption) throws IOException {
         final long size = source.toFile().length();
         log.info("{} copy process start [source:{}, destination :{}]", fileDescription, source, destination);
         final long start = System.currentTimeMillis();
-        Files.copy(source, destination, StandardCopyOption.COPY_ATTRIBUTES);
+        Files.copy(source, destination, standardCopyOption);
```

Comment:
Don't we want to preserve the vararg aspect of `Files.copy` here and have both `REPLACE_EXISTING` and `COPY_ATTRIBUTES` when copying the key ?
See https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/nio/file/Files.html#copy(java.nio.file.Path,java.nio.file.Path,java.nio.file.CopyOption...)

Reply:
it could be a good idea